### ISSUE TITLE
Indivisible (421170)

### ISSUE THREAD

**Issue by wfstuff** [ValveSoftware/Proton#5659]
# Compatibility Report
- Name of the game with compatibility issues: Indivisible
- Steam AppID of the game: 421170

## System Information
Steam Deck 64GB

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
I played Indivisible on my Steam Deck and until now I had no Problems (compatibility was unknown). I was running the default Linux version, I guess. The only problem I had, was that I could not change the resolution. It was fixed at 720p, but for the deck's screen it was okay. But right now, proton is used to run the game (compatibility has changed to yellow). The resolution is set to 800p, but I encountered several graphical glitches, whereas the native version seemed to run perfectly. I don't know if it's intended that the game runs under proton with graphical glitches, whereas the native version worked very well for me (except being locked to 720p). To be fair, I only played for like 2h and the native version might be buggy later on. Could it be a mistake that the game is running vie proton and not via the native version?

**Comment by kisak-valve** [comment]
Hello @wfstuff, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, if possible, please capture a screenshot of the render artifacts.

**Comment by wfstuff** [comment]
log file:
[steam-421170.log](https://github.com/ValveSoftware/Proton/files/8216256/steam-421170.log)

Screenshots:
The first batch is with the native linux version (on steam deck). The second batch is with the default (selected by valve) proton version. In the second batch there is also a black screen. At that point (or at least very near) the cut scene skips (some audio/dialogue is missing). Also the aspect ratio seems a little bit weird.

<details><summary>First screenshot batch</summary>

![20220309164142_1](https://user-images.githubusercontent.com/45985819/157482188-5ac57bfb-9264-43bf-b4a3-b88854570357.jpg)
![20220309164238_1](https://user-images.githubusercontent.com/45985819/157482207-004ac53f-55e7-42f3-b967-25ed64c42fb9.jpg)
![20220309164243_1](https://user-images.githubusercontent.com/45985819/157482213-e2ef6d1f-1cb2-4205-8ea1-c69b8d8765a1.jpg)
![20220309164301_1](https://user-images.githubusercontent.com/45985819/157482220-17026974-432c-410e-9ad3-cb585241d653.jpg)
![20220309164306_1](https://user-images.githubusercontent.com/45985819/157482228-0a8496db-6599-40a1-924d-e3609686b03a.jpg)
![20220309164322_1](https://user-images.githubusercontent.com/45985819/157482236-c536beb4-6f16-467b-ba74-498ff5e7bbed.jpg)
![20220309164329_1](https://user-images.githubusercontent.com/45985819/157482241-66b060f4-e3ae-4425-b064-e384e8a4e3b5.jpg)
![20220309164335_1](https://user-images.githubusercontent.com/45985819/157482249-47866e67-dab9-4f13-864a-46aaa1bf3b00.jpg)
![20220309164345_1](https://user-images.githubusercontent.com/45985819/157482255-f896bfc3-c3eb-4951-8185-847cee4f6091.jpg)
![20220309164414_1](https://user-images.githubusercontent.com/45985819/157482261-1ae45523-c0af-40c3-8bcf-1fd8f81d1aeb.jpg)

</details>

<details><summary>Second screenshot batch</summary>

![20220309164923_1](https://user-images.githubusercontent.com/45985819/157482267-61f8f719-e444-47f7-a078-2eeec9a24557.jpg)
![20220309164929_1](https://user-images.githubusercontent.com/45985819/157482275-663fa116-6517-45dc-9fa0-d063d82fcc10.jpg)
![20220309164944_1](https://user-images.githubusercontent.com/45985819/157482289-8372d03a-c121-482e-99ca-e265090406ff.jpg)
![20220309164949_1](https://user-images.githubusercontent.com/45985819/157482298-83b714c9-941c-44e7-92f0-b49c48ecfc08.jpg)
![20220309165004_1](https://user-images.githubusercontent.com/45985819/157482308-30480b12-3693-4f9c-8a3e-70e87a5be33f.jpg)
![20220309165011_1](https://user-images.githubusercontent.com/45985819/157482321-351d1210-9991-450f-a948-0a2ba4b15789.jpg)
![20220309165018_1](https://user-images.githubusercontent.com/45985819/157482326-d7b4e902-b28f-4989-bfcf-a6dba7faf19c.jpg)
![20220309165045_1](https://user-images.githubusercontent.com/45985819/157482335-7c493f98-48e1-4806-9f54-01c4a079cca0.jpg)

</details>

**Comment by Blisto91** [comment]
@wfstuff 
Does running the game with this launch option fix it? (For proton 7.0-1)
```
PROTON_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% 
```
If you are instead using proton experimental try
```
WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% 
```

**Comment by wfstuff** [comment]
Same problem (proton 7.0-1)

**Comment by cybik** [comment]
Hello, original port developer here. @kisak-valve the game has a native version, please force it back to native.

> @wfstuff Does running the game with this launch option fix it? (For proton 7.0-1)
> 
> ```
> PROTON_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% 
> ```
> 
> If you are instead using proton experimental try
> 
> ```
> WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% 
> ```

@blisto91 this shouldn't even be necessary. The native version should work fine.

I made it, and some others tested it already.

**Comment by Blisto91** [comment]
@cybik yes the native version should generally be preferred if there is one. And if it has issues should be brought up there to the devs 
I don't actually own the game so was more i was interested in if the above commands fixed the users proton issue.

Always good to see devs hanging around here tho 🙂

**Comment by cybik** [comment]
I am not here in any official capacity w.r.t. 505 Games, for the record.

**Comment by cybik** [comment]
I am still not here in any official capacity.

I recommend checking again :D

https://steamdb.info/app/421170/info/

**Comment by cybik** [comment]
@kisak-valve i think you can close this.

**Comment by ranplayer** [comment]
I've found a regression for this title: using Proton 9.0-4, videos/logos play fine, whereas in Proton 10.0-3 and Experimental 2025-11-26, they are rendered as colored bars or a black screen

Proton logs:  [steam-421170-proton103.tar.gz](https://github.com/user-attachments/files/24060935/steam-421170-proton103.tar.gz)

Main error lines:
```
0:00:36.019361847 21883 0x7585680b36d0 ERROR                  libav :0:: Invalid NAL unit size (332400358 > 10169).
0:00:36.019364973 21883 0x7585680b36d0 ERROR                  libav :0:: Error splitting the input into NAL units.
0:00:36.019393185 21883 0x758569442d30 ERROR                  libav :0:: Invalid NAL unit size (706329930 > 646).
0:00:36.019396371 21883 0x758569442d30 ERROR                  libav :0:: Error splitting the input into NAL units.
```

Specs:
- OS: Arch
- Kernel: 6.12.61
- GPU: RX 6700 XT (Mesa 25.2.7)
- CPU: Ryzen 7 5700G
- DE: Plasma 6.5.3 (wayland)
